securesign/sigstore-ocp
# Add Grafana operator for Monitoring

This pr is related to this [issue](https://issues.redhat.com/projects/SECURESIGN/issues/SECURESIGN-54?filter=allopenissues), and involves enabling monitoring via a grafana dashboard.

## Testing

1. Checkout and pull branch
2. Follow usual steps to deploy stack in the [README](https://github.com/securesign/sigstore-ocp#no-fail-quick-start-in-5-steps)
3.  Follow the steps described in [enable-grafana-monitoring.md](https://github.com/JasonPowr/sigstore-ocp/blob/d0c8ef14f97fdd4acc7456304fe83b597a1d647d/enable-grafana-monitoring.md)
4. Ensure that grafana gets installed and you can view the dashboards

## Comments
The dashboard in this pr completes everything defined in the jira + extra, If you believe that there should be more in the dashboard please feel free to reach out, or add it yourself.
